sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHost.cs

```diff
@@ -13,155 +13,124 @@
 
 namespace Microsoft.Azure.WebJobs.EventHubs.Processor
 {
-    internal class EventProcessorHost
+    internal class EventProcessorHost : EventProcessor<EventProcessorHostPartition>
```

Comment:
At some point, we probably want to consider renaming this, as `EventProcessorHost` specifically identifies a T1 type in the ecosystem and could be confusing.

Reply:
Yeah, there are so many things names *EventProcessor in the solution.